app/konnect/dev-portal/applications/dynamic-client-registration/custom.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Configure your Custom IdP for Dynamic Client Registration
+breadcrumb: Custom
+content_type: explanation
+description: Learn how to configure your custom IdP for Dynamic Client Registration in Konnect.
+---
+
+{{site.konnect_short_name}} Dev Portal supports a variety of the most widely adopted identity provider (IdP) for [Dynamic Client Registration (DCR)](/konnect/dev-portal/applications/dynamic-client-registration/):
+
+* Auth0
+* Azure
+* Curity
+* Okta
+
+If your third-party IdP is not on this list, you can still use your IdP with {{site.konnect_short_name}} by using Kong’s open source HTTP DCR bridge. This HTTP DCR bridge acts as a proxy and translation layer between your IdP and DCR applications in the Dev Portal. This bridge allows you to enable a DCR integration between the Dev Portal and your third-party IdP or IdP that isn't exposed to the internet. This integration can act as either a translation layer to bridge to your network <!--either translation layer or....???-->.
```

Comment:
@c3pko "This integration can act as either a translation layer to bridge to your network"

With the "either" in there, I think this sentence was meant to continue? What else can the integration act as other than the translation layer?